### ISSUE TITLE
S2D Default Locale Update

### ISSUE THREAD

**Issue by stsegovia-moz** [mozmeao/snippets-service#1032]
Hi Giorgos. I was testing out the SMS + Email send to device template for EN, I realized it wouldn't send me the email with the download link. I realize the reason is because the default locale for the S2D template is en-US, but in basket it's set as EN.

This disrepancy causes the snippet not to send. I've updated any published mobile snippet to have Locale as EN for the email trigger.

Can you update the default to be `EN` for the locale? Feel free to test this yourself.

TL;DR. Default locale should be `EN` not `en-US`. The `en-US` locale does not send the email.

---------

**Screenshot of Snippet Setting**
![Screen Shot 2019-06-10 at 3 12 14 PM](https://user-images.githubusercontent.com/47153345/59230385-27c1a900-8b92-11e9-9e07-adbdd4060ae5.png)

---------

**Basket for Firefox Movile**
![Screen Shot 2019-06-10 at 3 13 24 PM](https://user-images.githubusercontent.com/47153345/59230450-5a6ba180-8b92-11e9-913a-647e281c37a6.png)

**Comment by jkt230** [comment]
Just verifying if we can fix this soon? Thanks!

**Comment by stsegovia-moz** [comment]
As a temporary fix, I manually have been updating any EN Send to Device snippets. So I am just keeping that in mind when I make any new S2D snippets.

**Comment by glogiotatidis** [comment]
@stsegovia-moz in the title you mention `Country Code Update` but I guess you really mean locale. Working on this atm

**Comment by glogiotatidis** [comment]
Also updated to `locale = EN` 
 - https://snippets-admin.mozilla.org/admin/base/asrsnippet/11994/change/
 - https://snippets-admin.mozilla.org/admin/base/asrsnippet/11977/change/
 - https://snippets-admin.mozilla.org/admin/base/asrsnippet/11980/change/
 - https://snippets-admin.mozilla.org/admin/base/asrsnippet/9655/change/
 - https://snippets-admin.mozilla.org/admin/base/asrsnippet/11980/change/

**Comment by glogiotatidis** [comment]
Fixed in production

**Comment by glogiotatidis** [comment]
Note that this fix only changes the default when creating new snippets. I can always automated changing old snippets which are still `locale=en-US` upon request. All published snippets are using `locale=EN` now

**Comment by stsegovia-moz** [comment]
Thank you Giorgos! That's perfect. I don't think we need to worry about the unpublished snippets, since it's likely we won't re-use them any time soon.
